heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/KubernetesController.java

```diff
@@ -43,114 +49,110 @@ public KubernetesController(
    *
    * @return success
    */
-  public boolean killTopology() {
+  protected boolean killTopology() {
```

Comment:
It would be great to change the existing methods in this class to also return `void` and throw exceptions instead. I know that's scope creep on what you're adding in this PR though, so we could make that change in a follow up PR if you prefer.